packages/fonts/package.json

```diff
@@ -47,7 +47,7 @@
   },
   "main": "dist/index.js",
   "module": "dist-module/index.js",
-  "tyoes": "dist-types/index.d.ts",
+  "types": "dist-types/index.d.ts",
```

Comment:
This one stumped me quite a bit! It just kept refusing to load the types and I couldn't understand why... 🤦🤦🤦